src/seedu/addressbook/parser/Parser.java

```diff
@@ -17,8 +17,9 @@
 import seedu.addressbook.commands.DeleteCommand;
 import seedu.addressbook.commands.ExitCommand;
 import seedu.addressbook.commands.FindCommand;
+import seedu.addressbook.commands.SortCommand;
 import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
+import seedu.addressbook.commands.IncorrectCommand;	
```

Comment:
Is there white spaces after the semicolon? If yes, delete the white spaces since the checkstyle will detect the error later on.